.github/workflows/openqa_fullstack.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: openQA Fullstack
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+jobs:
+  fullstack:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/devel/openqa/ci/containers/base:latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Make project folder user-writable
+        run: sudo chown -R $USER ../../os-autoinst
```

Comment:
I wonder if this is necessary. Can you try without?

Reply:
It is, the directories are not writable by default.